lib/handle-pull-request-change.js

```diff
@@ -40,6 +42,11 @@ async function handlePullRequestChange (context) {
     allowMergeCommits,
     allowRevertCommits
   } = Object.assign({}, DEFAULT_OPTS, userConfig)
+
+  if (!enabled) {
+    return
```

Comment:
I'm not sure that this null return will take care of things. I think with this change as-is, the check will just hang indefinitely or time out and fail.

Instead, I think you'll need to update this to:

1. set `isSemantic`  to true when `enabled` is `false`
1. create a custom status description like "Check disabled in semantic.yml config`
1. Leave the call to `context.github.repos.createStatus()` intact

Reply:
@zeke Awesome, I'll get this updated this afternoon. Thank you!